torch/csrc/autograd/profiler.cpp

```diff
@@ -169,6 +173,14 @@ struct FileLineFunc {
   std::string funcname;
 };
 
+static std::atomic<size_t> corr_id_ {};
```

Comment:
nit: do we need another atomic? I thought we already had thread_id and thread-local number. Maybe we could combine them?

Reply:
@gdankel do we need global id here or thread local would be fine?
yeah, if we need global, i could prob. make it from thread id and local